lib/movement_problems/coordinate.py

```diff
@@ -0,0 +1,39 @@
+class Coordinate:
+    """
+        A class to calculate coordinates.
+        --Attributes
+            start_point : list
+                first point coordinate
+            end_point : list
+                second point coordinate
+        --Methods
+            get_point_by_percent_on_segment(percent)
+                takes in percent
+                :return coordinate of the percent on segment
+            divide_segment_into_equal_parts(parts):
+                takes in parts quantity, into which must be divided
+                :return coordinate of the parts on segment
+    """
+    def __init__(self, start_point, end_point):
```

Comment:
Please, specify somewhere, that y coordinates of those points should be the same.